### PR TITLE
Revised Intro to MTA 

### PR DESCRIPTION
**Pull request by RichardHoch**
Rewrite of Introduction to Migration Toolkit for Applications; revision of introduction to the UI guide.

Resolves https://issues.redhat.com/browse/WINDUP-3427 and https://issues.redhat.com/browse/TACKLE-857

Previews:
1. https://deploy-preview-573--windup-documentation.netlify.app/docs/getting-started-guide/master/index.html  Introduction to Migration Toolkit for Applications (complete)

2. https://deploy-preview-573--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#mta-about-console-guide_web-console-guide Intro to the User Interface guide, using the new language for MTA.  The rest of the UI is fixed in other PRs. I just want you to check the new intro.

### CODE REVIEW

File: docs/topics/about-the-web-console.adoc [hunk 1]
```diff
@@ -6,8 +6,11 @@
...
+
+ifdef::mtr[]
 The {WebName} for the {ProductName} allows a team of users to assess and prioritize migration and modernization efforts for a large number of applications. It allows you to group applications into projects for analysis and provides numerous reports that highlight the results.
+endif::[]
 
-ifndef::web-console-guide[]
-For more information on using the {WebName}, see the {ProductShortName} link:{ProductDocWebConsoleGuideURL}[_{WebConsoleBookName}_].
-endif::web-console-guide[]
+ifdef::mta[]
+The {WebName} for the {ProductName} lets you assess the risks involved in containerizing an application for hybrid cloud environments on Red Hat OpenShift or to analyze the changes that must be made in the code of an application in order to containerize the appliaction. It allows you to group applications into projects for analysis and provides numerous reports that highlight the results.
```
Comment (sshveta) [inline docs/topics/about-the-web-console.adoc:15]: Spelling : "Application"

**Review by sshveta — CHANGES_REQUESTED** [review]

File: docs/topics/about-the-web-console.adoc [hunk 2]
```diff
@@ -6,8 +6,11 @@
...
+
+ifdef::mtr[]
 The {WebName} for the {ProductName} allows a team of users to assess and prioritize migration and modernization efforts for a large number of applications. It allows you to group applications into projects for analysis and provides numerous reports that highlight the results.
+endif::[]
 
-ifndef::web-console-guide[]
-For more information on using the {WebName}, see the {ProductShortName} link:{ProductDocWebConsoleGuideURL}[_{WebConsoleBookName}_].
-endif::web-console-guide[]
+ifdef::mta[]
+The {WebName} for the {ProductName} lets you assess the risks involved in containerizing an application for hybrid cloud environments on Red Hat OpenShift or to analyze the changes that must be made in the code of an application in order to containerize the application. It allows you to group applications into projects for analysis and provides numerous reports that highlight the results.
```
Comment (rromannissen) [inline docs/topics/about-the-web-console.adoc:15]: The notion of projects is gone in MTA, this should be reviewed.

File: docs/topics/about-the-user-interface.adoc [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="about-the-user-interface_{context}"]
+= About the {WebName}
+
+The {WebName} for the {ProductName} allows a team of users to assess and analayze applications for risks and suitability for migration to hybrid cloud environments on Red Hat OpenShift.
+
+Use the {WebName} to assess and analyze your applications provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user interface.
```
Comment (rromannissen) [inline docs/topics/about-the-user-interface.adoc:11]: There seems to be something wrong with this sentence. Maybe replace "provides" with "and get"?

File: docs/topics/about-the-mta-operator.adoc [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+//
+// * docs/wgetting-started-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="about-the-mta-operator_{context}"]
+= About the {ProductShortName} Operator
+
+You can install the web console on OpenShift Container Platform versions 4.9-4.11 with the Migration Toolkit for Applications Operator.
+
+For installation instructions, see link:{ProductDocWebConsoleGuideURL}/index#installing-web-console-on-openshift_ocp-46[Installing the web console on OpenShift Container Platform 4.6 and later].
```
Comment (rromannissen) [inline docs/topics/about-the-mta-operator.adoc:11]: MTA doesn't support OCP 4.6, maybe wrong link?

File: docs/topics/mta-about-console-guide.adoc [hunk 5]
```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-about-console-guide_{context}"]
+= About the User Interface Guide
+
+This guide is for engineers, consultants, and others who want to use the {ProductName} ({ProductShortName}) {WebName} to accelerate large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user interface.
```
Comment (rromannissen) [inline docs/topics/mta-about-console-guide.adoc:9]: I would also add  "architects" to the list.

File: docs/topics/mta-about-the-intro-to-mta-guide.adoc [hunk 6]
```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-about-the-intro-to-mta-guide_{context}"]
+= About the {IntroToMTABookName}
+
+This guide is for engineers, consultants, and others who want to use the {ProductName} ({ProductShortName}) to accelerate large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. It provides an overview of the {ProductName} and how to get started using the tools to plan and run your migration.
```
Comment (rromannissen) [inline docs/topics/mta-about-the-intro-to-mta-guide.adoc:9]: I would add architects here as well

File: docs/topics/mta-what-is-the-toolkit.adoc [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+{ProductShortName} examines application artifacts, including project source directories and application archives, and then produces an HTML report highlighting areas needing changes. {ProductShortName} supports many migration paths including the following:
+
+* Upgrading to the latest release of Red Hat JBoss Enterprise Application Platform
+* Migrating from Oracle WebLogic or IBM WebSphere Application Server to Red Hat JBoss Enterprise Application Platform
+* Containerizing applications and making them cloud-ready
+* Migrating from Java Spring Boot to Quarkus
+* Upgrading from OpenJDK 8 to OpenJDK 11
+* Upgrading from OpenJDK 11 to OpenJDK 17
+* Migrating EAP Java applicatons to Azure
```
Comment (rromannissen) [inline docs/topics/mta-what-is-the-toolkit.adoc:32]: Azure App Service

File: docs/topics/mta-what-is-the-toolkit.adoc [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+* Migrating Spring Boot Java applications to Azure
```
Comment (rromannissen) [inline docs/topics/mta-what-is-the-toolkit.adoc:33]: Azure App Service

File: docs/topics/about-the-web-console.adoc (hunk 2 above)
Comment (RichardHoch) [reply docs/topics/about-the-web-console.adoc:15]: Done.

File: docs/topics/mta-about-console-guide.adoc (hunk 5 above)
Comment (RichardHoch) [reply docs/topics/mta-about-console-guide.adoc:9]: Done.

File: docs/topics/mta-about-the-intro-to-mta-guide.adoc (hunk 6 above)
Comment (RichardHoch) [reply docs/topics/mta-about-the-intro-to-mta-guide.adoc:9]: Done.

File: docs/topics/mta-what-is-the-toolkit.adoc (hunk 7 above)
Comment (RichardHoch) [reply docs/topics/mta-what-is-the-toolkit.adoc:32]: Done.

File: docs/topics/mta-what-is-the-toolkit.adoc (hunk 8 above)
Comment (RichardHoch) [reply docs/topics/mta-what-is-the-toolkit.adoc:33]: Done.

File: docs/topics/about-the-mta-operator.adoc (hunk 4 above)
Comment (RichardHoch) [reply docs/topics/about-the-mta-operator.adoc]: linked to installation chapter for now.

File: docs/topics/about-the-user-interface.adoc (hunk 3 above)
Comment (RichardHoch) [reply docs/topics/about-the-user-interface.adoc]: Fixed.

**Comment by RichardHoch** [comment]
@rromannissen I made the chnages you suggested. Please review this PR again.

**Review by rromannissen — APPROVED** [review]
LGTM!

File: docs/topics/about-the-web-console.adoc [hunk 9]
```diff
@@ -6,8 +6,11 @@
 [id="about-the-web-console_{context}"]
 = About the {WebName}
 
+
+ifdef::mtr[]
```
Comment (sshveta) [inline docs/topics/about-the-web-console.adoc:10]: mta

Comment (RichardHoch) [reply docs/topics/about-the-web-console.adoc:10]: No, MTR uses projects; MTA does not.

File: docs/topics/about-tools.adoc [hunk 10]
```diff
@@ -8,8 +8,18 @@
 
 The {ProductName} ({ProductShortName}) provides several tools to assist you in the various stages of your migration and modernization efforts. Review the details of each tool to determine which is right for your project.
 
-* CLI
+ifdef::mta[]
+* User interface
+endif::[]
+
+ifdef::mtr[]
```
[the remark this answers is not in the document]
Comment (RichardHoch) [reply docs/topics/about-tools.adoc:15]: No, that's the new MTA.

**Pull request merged**